assets/js/custom.js

```diff
@@ -1 +1,29 @@
 // Put your custom JS code here
+
+// self-hosting view.
+
+const storedSelfhost = localStorage.getItem('selfhost');
+
+const getSelfhost = () => {
+  if (storedSelfhost) {
+    return storedSelfhost;
+  }
+
+  return 'no';
+};
```

Comment:
Even though this function is only called once, I think it's clearest if we always try to fetch from localStorage inside the function: `return localStorage.getItem('selfhost') || 'no';`. Otherwise `getSelfhost` may not accurately reflect the correct state, depending on whether selfhost has been updated since initialization.